### PR TITLE
Support Net7.0 

### PR DESCRIPTION
**Pull request by sung-su**
This PR is for support net 7.0.
Can not recognized new workload manifest
in released dotnet sdk version 7.0.100-preview4.22252.9
```
$ dotnet workload install tizen
Workload ID tizen is not recognized.
```
There was no problem with the local built dotnet-sdk.
reported dotnet-sdk : https://github.com/dotnet/sdk/issues/25629

### CODE REVIEW

File: workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets [hunk 1]
```diff
@@ -45,6 +45,21 @@ Copyright (c) Samsung All rights reserved.
...
+      TargetingPackVersion="**FromWorkload**"
+      Profile="Tizen"
+    />
+    <KnownFrameworkReference
+      Include="Samsung.Tizen"
+      TargetFramework="net7.0-tizen$(_DefaultTargetPlatformVersion)"
+      TargetingPackName="Samsung.Tizen.Ref"
+      TargetingPackVersion="**FromWorkload**"
+      Profile="Tizen"
+    />
```
Comment (WonyoungChoi) [inline workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets:62]: Duplicated element.

File: workload/Config.mk [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 -include $(TMPDIR)/dotnet-version.config
 $(TMPDIR)/dotnet-version.config: $(TOP)/build/Versions.props
 	@mkdir -p $(TMPDIR)
-	@grep "<MicrosoftDotnetSdkInternalPackageVersion>" build/Versions.props | sed -e 's/<\/*MicrosoftDotnetSdkInternalPackageVersion>//g' -e 's/[ \t]*/DOTNET6_VERSION=/' > $@
-DOTNET6_VERSION_BAND = $(firstword $(subst -, ,$(DOTNET6_VERSION)))
+	@grep "<MicrosoftDotnetSdkInternalPackageVersion>" build/Versions.props | sed -e 's/<\/*MicrosoftDotnetSdkInternalPackageVersion>//g' -e 's/[ \t]*/DOTNET7_VERSION=/' > $@
+DOTNET7_VERSION_BAND = $(firstword $(subst -, ,$(DOTNET7_VERSION)))
 
-# DOTNET6_DESTDIR
+# DOTNET7_DESTDIR
```
Comment (WonyoungChoi) [inline workload/Config.mk:8]: I think it's better to remove the version number from the variable for later.

File: workload/Config.mk [hunk 3]
```diff
@@ -33,13 +33,14 @@ endif
...
+	ifneq ($(PULLREQUEST_ID),)
+		PRERELEASE_VERSION := ci.pr.gh$(PULLREQUEST_ID)
+	else
+		PRERELEASE_VERSION := ci.$(CURRENT_BRANCH)
+	endif
 endif
 
-TIZEN_WORKLOAD_VERSION_FULL := $(TIZEN_WORKLOAD_VERSION)
+TIZEN_WORKLOAD_VERSION_FULL := $(TIZEN_WORKLOAD_VERSION)-$(PRERELEASE_VERSION).$(TIZEN_COMMIT_DISTANCE)
+#TIZEN_WORKLOAD_VERSION_FULL := $(TIZEN_WORKLOAD_VERSION)
```
Comment (WonyoungChoi) [inline workload/Config.mk:46]: Please add EOF

File: workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets (hunk 1 above)
Comment (JoonghyunCho) [reply workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets:62]: yes, I will remove and update `$(_DefaultTargetPlatformVersion)` to `7.0` (hard coded value)

File: workload/Config.mk (hunk 2 above)
Comment (JoonghyunCho) [reply workload/Config.mk:8]: agreed!

**Review by JoonghyunCho — APPROVED** [review]
LGTM.
We still need to follow up the install fail issue.

**Pull request merged**